Source/String.swift

```diff
@@ -68,7 +68,7 @@ extension String {
     }
 
     public func trimLeft(_ characterSet: CharacterSet) -> String {
-        var start = characters.count
+        var start = 0
```

Comment:
While this probably doesn't break anything, it's still not the direct equivalent. Should probably be undone.